python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -373,251 +375,246 @@ def makePlot(
         # zoomed-in maps.
         histColors = ["#265D40", "#8B0000", "#00008B"]
 
+        mapName = data["data"].ref.datasetType.name
+        mapData = data["data"].get()
+
         with rc_context(rcparams):
-            for mapName, mapDataHandle in data.items():
-                mapData = mapDataHandle.get()
-                fig, ((ax1, ax2), (ax3, ax4)) = plt.subplots(2, 2, figsize=(16, 16))
-
-                # Reduce whitespace but leave some room at the top for
-                # plotInfo.
-                plt.subplots_adjust(left=0.064, right=0.96, top=0.855, bottom=0.07, wspace=0.18, hspace=0.24)
-
-                # Get the values for the valid pixels of the full tract.
-                values = mapData[mapData.valid_pixels]
-                goodValues = np.isfinite(values)
-                values = values[goodValues]  # As a precaution.
-
-                # Make a concise human-readable label for the plot.
-                plotInfo["unit"] = "N/A"  # Unless overridden.
-                if hasattr(mapData, "metadata") and all(
-                    key in mapData.metadata for key in ["DESCRIPTION", "OPERATION", "UNIT"]
-                ):
-                    hasMetadata = True
-                    metadata = mapData.metadata
-                    plotInfo["description"] = metadata["DESCRIPTION"]
-                    plotInfo["operation"] = metadata["OPERATION"]
-                    if metadata["UNIT"]:
-                        plotInfo["unit"] = metadata["UNIT"]
-                    elif metadata["UNIT"] == "":
-                        plotInfo["unit"] = "dimensionless"
-                else:
-                    hasMetadata = False
-                    plotInfo["operation"] = getLongestSuffixMatch(
-                        mapName, ["min", "max", "mean", "weighted_mean", "sum"]
-                    ).replace("_", " ")
-                plotInfo["coaddName"] = mapName.split("Coadd_")[0]
-                plotInfo["operation"] = (
-                    plotInfo["operation"].replace("minimum", "min").replace("maximum", "max")
-                )
-                propertyName = mapName[
-                    len(f"{plotInfo['coaddName']}Coadd_") : -len(plotInfo["operation"])
-                ].strip("_")
-                if not hasMetadata:
-                    # Infer the property description from the map name (all
-                    # lower case), and properly handle formatting.
-                    plotInfo["description"] = (
-                        propertyName.replace("_", " ")
-                        .replace("psf", "PSF")
-                        .replace("dcr", "DCR")
-                        .replace("dra", r"$\Delta$RA")
-                        .replace("ddec", r"$\Delta$Dec")
-                    )
-                plotInfo["property"] = (
+            fig, ((ax1, ax2), (ax3, ax4)) = plt.subplots(2, 2, figsize=(16, 16))
```

Comment:
Can we either remove the uses of plt on this ticket or make a new ticket to remove them? As discussed on the previous ticket they are very slow.

Reply:
Yes, we have a new ticket ([DM-47962](https://rubinobs.atlassian.net/browse/DM-47962)) to remove the uses of plt. I discussed this with @erykoff. For now, the plan is to leave the slow plt code in and address its removal in [DM-47962](https://rubinobs.atlassian.net/browse/DM-47962) as soon as the new version of SkyProj becomes available, likely by January.

Reply:
Eli checked and confirmed that the fix does work with the *new* version.